### PR TITLE
Add LDBC test to cover precision loss

### PR DESCRIPTION
**Pull request by Mats-SX**
This is the query that originally exposed the precision loss bug.

### CODE REVIEW

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/LdbcAcceptanceTest.scala [hunk 1]
```diff
@@ -38,4 +38,25 @@ class LdbcAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerTestSupp
...
+        |ORDER BY count DESC, friendId ASC
+        |LIMIT {3}
+      """.stripMargin
+
+    val params: Map[String, Any] = Map("1" -> 0, "2" -> 1, "3" -> 10)
+
+    val result = executeWithAllPlanners(s"PROFILE $updatedLdbc12", params.toSeq:_*)
+    // no precision loss resulting in insane numbers
+    result.executionPlanDescription().toString should not include "305862903934218048"
+    result.executionPlanDescription().toString should not include "9223372036854775807"
```
Comment (thobe) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/LdbcAcceptanceTest.scala]: These assertions look very specific. There are plenty other "insane numbers", I don't think listing every imaginable "insane number" is a good strategy for a test.

Comment (Mats-SX) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/LdbcAcceptanceTest.scala]: That's why I only wrote down two... ;)

I could also assert that the plan does not include `NodeHashJoin`, but that would also be weird. What I wanted to do is to make an assertion on the `estimatedRows` column, but I don't know how to do that. Do you have an idea?

Comment (thobe) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/LdbcAcceptanceTest.scala]: I think you should only assert on the effect. Which in this case would be that there is no `NodeHashJoin` in the plan.

If you do want to perform assertions on the numbers, you should get them out as numbers and assert that they are in a valid range.

Comment (pontusmelke) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/LdbcAcceptanceTest.scala]: I agree , either check for `NodeHashJoin` or `estimatedRows < smallestNumberThatIsStillInstane`. Maybe it is possible to do it as a `CardinalityModel` test?

**Comment by Mats-SX** [comment]
@thobe @pontusmelke How about this?

**Comment by thobe** [comment]
Much better, @Mats-SX.

**Comment by pontusmelke** [comment]
:+1:

**Pull request merged**